/**:
  ros__parameters:
    robot_base_frame: base_link
    costmap_topic: map
    costmap_updates_topic: map_updates
    visualize: true
    planner_frequency: 0.15
    progress_timeout: 30.0
    potential_scale: 3.0
    orientation_scale: 0.0
    gain_scale: 1.0
    transform_tolerance: 0.3
    min_frontier_size: 0.75